### PR TITLE
Adding Contact section

### PR DESCRIPTION
**Pull request by SnakyMz**
Things I have done:

- [ ] Adding Contact form section
- [ ] Adding Validation and rules to form input
- [ ] Adding Formspree service to form input

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,6 +152,21 @@ <h2 class="outro-head">About Myself</h2>
...
+                <h2 class="form-head">Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a
+                    project that needs coding. I’d love to help with it!</p>
+                <form action="https://formspree.io/f/mayzgpbp" method="POST">
+                    <input type="text" name="fullname" maxlength="30" placeholder="Lorem Ipsum" required>
+                    <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                    <textarea name="message" id="message" cols="30" maxlength="500" rows="7"
+                        required>Write your message here...</textarea>
+                    <button type="submit" id="submit" class="see-project">Get in touch</button>
+                </form>
```
Comment (Eternalgratis) [inline index.html]: - Currently, all the input fields are not responding in one attempt. You'll need to try several times before it responds to input the required data. Kindly fix this issue as it is not good for the user experience.

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
Hi @SnakyMz @Godedict , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there

## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ Linter error checks validated


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (SnakyMz) [reply index.html]: I had justify content and align items in the section tag, which prevented the interaction. Big thanks for the review

**Review by Zuhaib042 — APPROVED** [review]
Hi @SnakyMz 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Fixed the bug reported by the previous reviewer
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**